sdk/src/main/java/com/silanis/esl/sdk/examples/AccountLimitSupportedLanguagesSettingsExample.java

```diff
@@ -0,0 +1,38 @@
+package com.silanis.esl.sdk.examples;
+
+import com.silanis.esl.sdk.SupportedLanguages;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class AccountLimitSupportedLanguagesSettingsExample extends SDKSample{
+
+    public SupportedLanguages  patchedLimitSupportedLanguagesSettings1, patchedLimitSupportedLanguagesSettings2;
+
+    public static void main(String... args) {
+        new AccountEmailReminderSettingsExample().run();
+    }
+    @Override
+    protected void execute() {
+        SupportedLanguages supportedLanguages1 = getSupportedLanguages(Arrays.asList("en", "fr"), "fr");
+        SupportedLanguages supportedLanguages2 = getSupportedLanguages(Arrays.asList("en", "de"), "de");
+
```

Comment:
Instead of initialising supported languages resource, can you make use of the builder ?

Example:
     SupportedLanguages supportedLanguages2 = AccountLimitSupportedLanguagesSettingsBuilder
                .NewLimitSupportedLanguagesSettings()
                .WithDefaultSignerLanguage("de")
                .WithSignerLanguages(supportedLanguagesList2)
                .Build();